src/status_im/transactions/subs.cljs

```diff
@@ -1,25 +1,25 @@
 (ns status-im.transactions.subs
   (:require-macros [reagent.ratom :refer [reaction]])
-  (:require [re-frame.core :refer [register-sub subscribe]]
+  (:require [re-frame.core :refer [reg-sub-raw subscribe]]
             [status-im.utils.hex :as i]))
 
-(register-sub :transactions
+(reg-sub-raw :transactions
```

Comment:
For this first step I focused on dumb migration. `reg-sub` looks better but involves some code changes.
I plan to list some best-practises before introducing it.